apps/files/src/components/FileSharingSidebar.vue

```diff
@@ -133,27 +136,21 @@ export default {
     }
   },
   computed: {
-    ...mapGetters('Files', ['selectedFiles', 'shareOpen', 'shares', 'sharesError', 'sharesLoading']),
+    ...mapGetters('Files', ['selectedFiles', 'shareOpen', 'shares', 'sharesError', 'sharesLoading', 'davProperties']),
     ...mapState(['user']),
     $_ocCollaborationStatus_autocompletePlacholder () {
       return this.$gettext('Add name(s), email(s) or federation ID\'s')
     },
     owner () {
-      if (this.shares.length === 0) {
-        return {
-          avatar: 'https://picsum.photos/64/64?image=1074',
-          name: this.user.id,
-          displayName: this.user.displayname
-        }
-      }
-      const s = this.shares[0]
-      return {
-        avatar: 'https://picsum.photos/64/64?image=1074',
-        name: s.uid_owner,
-        displayName: s.info.displayname_owner
-      }
-    }
+      this.$client.files.fileInfo(this.selectedFiles[0].path, this.davProperties).then(res => {
+        let ownerId = res['fileInfo']['{http://owncloud.org/ns}owner-id']
+        this.ownerAvatar = `${this.$root.config.server}/remote.php/dav/avatars/${ownerId}/128.png`,
```

Comment:
computed methods should only return a value, not set them: `vue/no-side-effects-in-computed-properties`
These are being re-evaluated as the data model changes, so they may have undesired side-effects and rund multiple times.

Reply:
https://github.com/vuejs/eslint-plugin-vue/blob/master/docs/rules/no-side-effects-in-computed-properties.md

Reply:
Fixed now